pkg/cmd/admin/policy/cani.go

```diff
@@ -79,6 +83,7 @@ func NewCmdCanI(name, fullName string, f *clientcmd.Factory, out io.Writer) *cob
 	cmd.Flags().StringVar(&o.User, "user", o.User, "Check the specified action using this user instead of your user.")
 	cmd.Flags().StringSliceVar(&o.Groups, "groups", o.Groups, "Check the specified action using these groups instead of your groups.")
 
+	kcmdutil.AddPrinterFlags(cmd)
```

Comment:
@fabianofranz from what I tested manually, it does not crash, but the same time it takes no effect. I'm not sure if we should worry about it.